src/components/gravidkrav/GravidKravReducer.ts

```diff
@@ -20,7 +21,7 @@ const GravidKravReducer = (state: GravidKravState, action: GravidKravAction): Gr
 
     case Actions.Fra:
       if (payload?.fra === undefined) {
-        nextState.til = undefined;
+        nextState.fra = undefined;
       } else {
         nextState.fra = parseDateTilDato(payload?.fra);
```

Comment:
Kan vurdere å lage en ternary for økt lesbarhet..

nextState.fra = payload?.fra ? parseDateTilDato(payload?.fra) : undefined